index.html

```diff
@@ -0,0 +1,127 @@
+              <h3 class="burger_price">6,500원</h3>
+              <p class="desc">얼마나 맛있게요?</p>
+            </div>
+          </article>
+          <article>
+            <img
+              src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1303.png?"
+              alt="Mcspicy shanghai burger"
+            />
+            <div class="burger_content">
+              <h2 class="burger_name">맥스파이시 상하이 버거</h2>
+              <h3 class="burger_price">5,600원</h3>
+              <p class="desc">얼마나 맛있게요?</p>
+            </div>
+          </article>
+          <article>
+            <img
+              src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/6312.png?"
+              alt="egg bulgogi burger"
+            />
+            <div class="burger_content">
+              <h2 class="burger_name">에그 불고기 버거</h2>
+              <h3 class="burger_price">4,000원</h3>
+              <p class="desc">얼마나 맛있게요?</p>
+            </div>
+          </article>
+          <article>
+            <img
+              src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1433.png?"
+              alt="shoobee burger"
+            />
+            <div class="burger_content">
+              <h2 class="burger_name">슈비 버거</h2>
+              <h3 class="burger_price">6,300원</h3>
+              <p class="desc">얼마나 맛있게요?</p>
+            </div>
+          </article>
+          <article>
+            <img
+              src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1314.png?"
+              alt="bacon tomato deluxe"
+            />
+            <div class="burger_content">
+              <h2 class="burger_name">베이컨 토마토 디럭스</h2>
+              <h3 class="burger_price">6,300원</h3>
+              <p class="desc">얼마나 맛있게요?</p>
+            </div>
+          </article>
+          <article>
+            <img
+              src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/6310.png?"
+              alt="cheese burger"
+            />
+            <div class="burger_content">
+              <h2 class="burger_name">치즈버거</h2>
+              <h3 class="burger_price">3,100원</h3>
+              <p class="desc">얼마나 맛있게요?</p>
+            </div>
+          </article>
+          <article>
+            <img
+              src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/6169.png?"
+              alt="bigmac"
+            />
+            <div class="burger_content">
+              <h2 class="burger_name">맥치킨</h2>
+              <h3 class="burger_price">4,100원</h3>
+              <p class="desc">얼마나 맛있게요?</p>
+            </div>
+          </article>
+        </section>
+        <aside class="cart_container">
+          <div class="cart_section">
+            <h3>장바구니</h3>
+          </div>
+          <div class="cart_section total_price">
+            <h3>누적 금액</h3>
+            <div>0원</div>
+          </div>
```

Comment:
갠적으로 장바구니 태그들을 차라리 하나로 묶는 게 좋을 것 같아요
```suggestion
          <div class="cart_section">
              <h3>장바구니</h3>
              <strong>누적 금액</strong>
              <p>0원</p>
           </div>
```
.cart_section이 중복되니까 하나로 묶고, 
첨에 .total_price div 에 누적금액 스타일링을 해주셨는데 `.cart_section > p` 로 접근하거나
p 태그에 class를 부여해서 `.cart_section > .total_price` 등의 방법으로 스타일링하면 더 알아보기 쉬울 것 같아요
  
저도 시멘틱 태그 사용이 아직 미숙하지만 위와 같이 구현할 수도 있을 것 같아요~..

Reply:
굳이 누적금액과 0원을 묶은 이유는, 장바구니(제목)와 누적금액/0원 부분의 스타일링(margin-bottom)이 공통되어서 코드 중복을 피하기 위해 그렇게 지정했습니다. 그렇지만 내용으로 보면 확실히 장바구니 글씨 부분을 전체의 제목으로 볼 수 있을 것 같네요. 시멘틱 태그의 측면에서는 이 편이 나은 것 같습니다. 코드를 중복하지 않으면서 시멘틱 태그를 잘 쓸 수 있는 방법이 있을 지 한번 고민해볼게요!